about.html

```diff
@@ -12,36 +12,18 @@
     We're Hackatron.
   </p>
 
-  <h2 id="where">Where</h2>
-  <div class="block">
-    <div class="venue">
-      <h3><a href="http://www.fungostudios.com/">FunGo Studios</a></h3>
-      <p>
-        Via Sile, 41<br/>
-        31056 Roncade<br/>
-        Treviso, Italy
-      </p>
-    </div>
-    <div class="venue">
-      <h3><a href="http://memecoworking.com/">Meme Coworking</a></h3>
-      <p>
-        Via Marignana, 86<br/>
-        31020 Mogliano Veneto<br/>
-        Treviso, Italy
-      </p>
-    </div>
-  </div>
```

Comment:
@potomak FunGo Studios doesn't exist anymore, and we are no more in contact with Meme Coworking. Usually we meet at h-farm but since there is no more recurrent events, I would remove the where section.